src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -16874,12 +16865,12 @@ static void F(B<object?> b1, B<object> b2)
                 source,
                 parseOptions: TestOptions.Regular8);
             comp.VerifyDiagnostics(
-                // (11,14): warning CS8619: Nullability of reference types in value of type 'A<object?>' doesn't match target type 'A<object>'.
+                // (11,14): warning CS8619: Nullability of reference types in value of type 'B<object?>' doesn't match target type 'A<object>'.
```

Comment:
>// (11,14): warning CS8619: Nullability of reference types in value of type 'B<object?>' doesn't match target type 'A<object>'. [](start = 16, length = 127)

It looks like the warning is inaccurate and confusing. I believe the original warning was better and correct.